Podfile

```diff
@@ -11,7 +11,7 @@ workspace 'WordPress.xcworkspace'
 ##
 def wordpress_shared
     ## for production:
-    pod 'WordPressShared', '~> 1.8.13'
+    pod 'WordPressShared', :git => 'https://github.com/wordpress-mobile/WordPress-iOS-Shared.git', :commit  => 'efe5a065f3ace331353595ef85eef502baa23497'
```

Comment:
Leaving this here so reviewers can see Danger failing the build without having to push new commits.

If I'll end up merging this PR on this experimental repo, I'll remove it.